Dockerfile

```diff
@@ -26,10 +26,10 @@ RUN apk add --no-cache ${ALPINE_PKG_BASE} ${ALPINE_PKG_EXTRA}
 
 WORKDIR /app
 
-COPY . .
+COPY go.mod vendor* ./
 RUN [ ! -d "vendor" ] && go mod download all || echo "skipping..."
 
-RUN go mod tidy
```

Comment:
I am not sure why go mod tidy was still in this file. I think it should be removed correct?